src/services/GraphInit.jl

```diff
@@ -239,6 +239,20 @@ function initVariable!( dfg::AbstractDFG,
   return nothing
 end
 
+function initVariable!( dfg::AbstractDFG, 
+                      label::Symbol, 
+                      samplable_belief,
+                      solveKey::Symbol=:default;
+                      dontmargin::Bool=false,
+                      N::Int=getSolverParams(dfg).N  )
+  #
+  variable = getVariable(dfg, label)
+  M = getManifold(variable)
+  points = [samplePoint(M, samplable_belief) for _ = 1:N]
```

Comment:
does samplePoint work in all cases?  is that from IIF or MJL?

Reply:
It's from IIF, I would like it to work in all cases. 
What is different here than your suggestion is it works for containers such as Distributions.jl. 
We can add overloads for specifying the mean and covariance, but I thought it would be nicer to allow any distribution you want.